compiler/test/dotc/tests.scala

```diff
@@ -177,6 +178,8 @@ class tests extends CompilerTest {
   @Test def neg_all = compileFiles(negDir, verbose = true, compileSubDirs = false)
   @Test def neg_typedIdents() = compileDir(negDir, "typedIdents")
 
+  @Test def negVarargsT1625 = compileFiles(negDir + "varargsInMethodsT1625/")
```

Comment:
This is an unnecessary complication, just put your test sources in `tests/neg/` and remove this `@Test def ...`. Also use the prefix `i1625` for the test sources.